samples/app-hello-world/python/README.md

```diff
@@ -0,0 +1,222 @@
+---
+page_type: sample
+products:
+- office-365
+languages:
+- python
+title: Microsoft Teams python Helloworld Sample
+description: Explore a comprehensive Microsoft Teams hello world sample app built with Node.js, demonstrating key features such as tabs, bots, and messaging extensions.
+extensions:
+  contentType: samples
+  createdDate: 10/19/2022 10:02:21 PM
+urlFragment: officedev-microsoft-teams-samples-app-hello-world-python
+---
+
+# Microsoft Teams hello world sample app.
+
+- This Hello World sample app showcases the fundamental features of Microsoft Teams, including tabs, bots, and messaging extensions, all built with Node.js.
+
```

Comment:
Here its mentioned as NodeJS, this is a python sample update description

Reply:
fixed